### ISSUE TITLE
2.9.1 Patch release plan

### ISSUE THREAD

**Issue by joeparsons** [az-digital/az_quickstart#3144]
- #3072
- #3031
- #3058
- #2594
- #3138

## Pick list
```
git cherry-pick 35d86c8974fb45a5c29895b6f9b73ad2a15dc218
git cherry-pick 829f16d0076e9be7eda253a413c08f0d8b58ddbe
git cherry-pick 92bcc906f9b85d49d0960c77bd561a0c374393fa
git cherry-pick fa708e5ed8d0e1386ad49860e1ed2a3f21793995
git cherry-pick 858b2ec310c274acc058d13c42d44bfbd4c23cad
```

**Comment by joeparsons** [comment]
## Release notes snippet
`git log 2.9.0..HEAD --pretty='format:- %s'`
```
- Closes #1814, Closes #1991 Hiding Events from views currently references start date, should reference end date (#3138)
- Closes #2550 Add support for future dates by setting publication status to "forthcoming" (#2594)
- Fixes #3012 Fixes #3119 Digital Object Identifiers to hyperlinks (#3058)
- Fixes #3030 Link Publication Titles to Node. (#3031)
- Closes #3060 - Fix CKEditor editor experience to match previous version (#3072)
- Back to dev.
```

**Comment by joeparsons** [comment]
https://github.com/az-digital/az_quickstart/releases/tag/2.9.1
